manifests/components/versions.json

```diff
@@ -1,11 +1,11 @@
 {
-    "cert-manager": "bitnami/cert-manager:0.5.2-r22",
-    "external-dns": "bitnami/external-dns:0.5.4-r8",
-    "elasticsearch": "bitnami/elasticsearch:5.6.12-r2",
-    "fluentd": "bitnami/fluentd:1.2.2-r22",
-    "grafana": "bitnami/grafana:5.3.4-r6",
-    "kibana": "bitnami/kibana:5.6.12-r15",
-    "nginx-ingress-controller": "bitnami/nginx-ingress-controller:0.21.0-r0",
+    "cert-manager": "bitnami/cert-manager:0.5.2-r37",
+    "external-dns": "bitnami/external-dns:0.5.9-r19",
+    "elasticsearch": "bitnami/elasticsearch:5.6.14-r2",
+    "fluentd": "bitnami/fluentd:1.3.2-r13",
+    "grafana": "bitnami/grafana:5.4.2-r12",
+    "kibana": "bitnami/kibana:5.6.14-r11",
+    "nginx-ingress-controller": "bitnami/nginx-ingress-controller:0.21.0-r12",
     "oauth2_proxy": "bitnami/oauth2-proxy:0.20180625.74543-debian-9-r6",
```

Comment:
The latest available version for oauth2-proxy image is `bitnami/oauth2-proxy:0.20180625.74543-r107`.

Reply:
Right. Done.